source-git/workflow.org

```diff
@@ -0,0 +1,65 @@
+* High-level workflow
+** Source-git repositories are forks of the upstream repo
+** Configuration is added to each downstream branch in the ~.distro~ directory
+** Each source-git repo maps to one or more dist-git repositories
+** Each downstream branch maps to one or more branches in dist-git
+** Configuration tells tooling:
+*** The commit after which downstream changes start (the last upstream commit)
+*** The way to get the source-archive from the last upstream commit
+*** The way to transform commits since the last upstream commit to patches
+*** The way to update the spec-file
+**** Version and release fields
+**** Source-tags
+**** Patch-tags
+**** Prep-section
+** Transforming source-git to dist-git is customizable
+** Each distribution has its own space to store source-git repos
```

Comment:
I wonder if we should comment here about projects which want to host source-git repos in locations of their choice.